### PR TITLE
Create Ask for more details.js

### PR DESCRIPTION
**Pull request by mjonssonme**
In the spirit of :trebek we also need to be able to ask for more information, using !huh

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (earlduque) [inline .vscode/settings.json:1]: remove this from PR

File: Parsers/Give a Magic 8Ball reaction.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+/*
```
Comment (earlduque) [inline Parsers/Give a Magic 8Ball reaction.js:1]: remove this from PR

File: Parsers/Ask for more details.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+/*
```
Comment (earlduque) [inline Parsers/Ask for more details.js:1]: If this is supposed to be a react parser then all you need is
```js
/*
emoji:huh
*/

**Review by earlduque — CHANGES_REQUESTED** [review]
@mjonssonme Hey Mattias, before I approve this, can you remove the irrelevant files that do not pertain to this PR? Additionally, if you meant it to be in the spirit of +:trebek: did you want to change the template to match a `react parser` instead of a `chat` parser? Check out the trebek parser's file headers to see what I mean

File: Parsers/Give a Magic 8Ball reaction.js (hunk 2 above)
Comment (mjonssonme) [reply Parsers/Give a Magic 8Ball reaction.js]: done

**Review by earlduque — APPROVED** [review]

**Pull request merged**